django/forms/templates/django/forms/attrs.html

```diff
@@ -0,0 +1 @@
+{% for name, value in attrs.items %}{% if value is not False %} {{ name }}{% if value is not True %}="{{ value|stringformat:'s' }}"{% endif %}{% endif %}{% endfor %}
```

Comment:
GitHub has helpfully pointed out that this template is the only one without a newline at the end of the file. Hardly worth fixing except for consistency with the others.

Reply:
This is deliberate, see https://github.com/django/django/pull/14819#discussion_r701334868

There's also a test added to make sure this doesn't regress.

Reply:
Ah, my apologies for wading in late :) I didn't see that one in the substantial outdated/resolved list. Please ignore me :)

Reply:
It's a good call out. I wonder if there are more templates we need to use this work around. I think not (they have plenty of line breaks, so I don't see one at the end of the file making much difference) but ...